test/e2e/feature/feature.go

```diff
@@ -23,116 +23,339 @@ import (
-	InPlacePodVerticalScaling               = framework.WithFeature(framework.ValidFeatures.Add("InPlacePodVerticalScaling"))
-	IPv6DualStack                           = framework.WithFeature(framework.ValidFeatures.Add("IPv6DualStack"))
-	Kind                                    = framework.WithFeature(framework.ValidFeatures.Add("Kind"))
-	KubeletCredentialProviders              = framework.WithFeature(framework.ValidFeatures.Add("KubeletCredentialProviders"))
-	KubeletSecurity                         = framework.WithFeature(framework.ValidFeatures.Add("KubeletSecurity"))
-	KubeProxyDaemonSetDowngrade             = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetDowngrade"))
-	KubeProxyDaemonSetUpgrade               = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetUpgrade"))
-	KubeProxyDaemonSetMigration             = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetMigration"))
-	LabelSelector                           = framework.WithFeature(framework.ValidFeatures.Add("LabelSelector"))
-	LocalStorageCapacityIsolation           = framework.WithFeature(framework.ValidFeatures.Add("LocalStorageCapacityIsolation"))
-	LocalStorageCapacityIsolationQuota      = framework.WithFeature(framework.ValidFeatures.Add("LocalStorageCapacityIsolationQuota"))
-	MasterUpgrade                           = framework.WithFeature(framework.ValidFeatures.Add("MasterUpgrade"))
-	MemoryManager                           = framework.WithFeature(framework.ValidFeatures.Add("MemoryManager"))
-	NEG                                     = framework.WithFeature(framework.ValidFeatures.Add("NEG"))
-	NetworkingDNS                           = framework.WithFeature(framework.ValidFeatures.Add("Networking-DNS"))
-	NetworkingIPv4                          = framework.WithFeature(framework.ValidFeatures.Add("Networking-IPv4"))
-	NetworkingIPv6                          = framework.WithFeature(framework.ValidFeatures.Add("Networking-IPv6"))
-	NetworkingPerformance                   = framework.WithFeature(framework.ValidFeatures.Add("Networking-Performance"))
-	NetworkPolicy                           = framework.WithFeature(framework.ValidFeatures.Add("NetworkPolicy"))
-	NodeAuthenticator                       = framework.WithFeature(framework.ValidFeatures.Add("NodeAuthenticator"))
-	NodeAuthorizer                          = framework.WithFeature(framework.ValidFeatures.Add("NodeAuthorizer"))
-	NodeLogQuery                            = framework.WithFeature(framework.ValidFeatures.Add("NodeLogQuery"))
-	NodeOutOfServiceVolumeDetach            = framework.WithFeature(framework.ValidFeatures.Add("NodeOutOfServiceVolumeDetach"))
-	NoSNAT                                  = framework.WithFeature(framework.ValidFeatures.Add("NoSNAT"))
+	// Please keep the list in alphabetical order.
+
+	// TODO: document the feature (owning SIG, when to use this feature for a test)
```

Comment:
Documenting "when to use this feature" has two purposes:
- making it clear when to reuse an existing label for a new test
- together with the owning SIG, make it clear which jobs should test the feature and how it needs to be configured

Reply:
The "how it needs to be configured" might have to be called out explicitly?